filter/index.js

```diff
@@ -1,25 +1,22 @@
 function onlyEven(array) {
-  const evens = [];
-  for (let i = 0; i < array.length; i++) {
-    if (array[i] % 2 === 0) {
-      evens.push(array[i]);
-    }
-  }
-  return evens;
+  return array.filter(n => n % 2 === 0)
 }
 
-// refactor with filter!! OMG!
 
 function onlyOneWord(array) {
-  //where are the snacks?
+  return array.filter(n => !n.includes(" "))
 }
 
 function positiveRowsOnly(array) {
-  //is Dan human?
+  return array.filter(n => n.some(y => y < 0)===false)
 }
```

Comment:
Try declaring a falsy statement initially with a `!` in front of the n.some instead of after.
```js
return array.filter(x => !x.some(y => y < 0)
)```